### PR TITLE
optimize: extract redundancy code

### PR DESCRIPTION
**Pull request by heyaping388**
### Ⅰ. Describe what this PR did
redundancy code optimization

### CODE REVIEW

File: core/src/main/java/io/seata/core/rpc/processor/server/AbstractRemotingProcessor.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import io.netty.channel.ChannelHandlerContext;
+import io.seata.core.rpc.processor.RemotingProcessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author xhe 774630093@qq.com
+ * @classname AbstractRemotingProcessor
+ * @description AbstractRemotingProcessor
+ * @date 2020/10/10 16:22
```
Comment (jsbxyyx) [inline core/src/main/java/io/seata/core/rpc/processor/server/AbstractRemotingProcessor.java:27]: remove tag

**Comment by a364176773** [comment]
add optimize: before the heading

**Review by l81893521 — COMMENTED** [review]
Please change title begin with "optimize: "

File: core/src/main/java/io/seata/core/rpc/processor/server/AbstractRemotingProcessor.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            if (LOGGER.isInfoEnabled()) {
+                LOGGER.info("closeChannelHandlerContext channel:" + ctx.channel());
+            }
+            ctx.disconnect();
+            ctx.close();
+        } catch (Exception exx) {
+            LOGGER.error(exx.getMessage());
+        }
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info(String.format("close a unhandled connection! [%s]", ctx.channel().toString()));
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/processor/server/AbstractRemotingProcessor.java:44]: LOGGER.info("close a unhandled connection! [{}]", ctx.channel().toString());

**Review by xingfudeshi — COMMENTED** [review]
Thanks!I left a comment.

**Comment by l81893521** [comment]
This pr was freezing for a long time. So I will close it and waiting for you next contribution.

**Pull request closed without merging**